platform/xbox/functions/math/nexttoward.c

```diff
@@ -1,20 +1,108 @@
 #include <math.h>
 #include <assert.h>
 
+// Adapted from public domain code by Danny Smith <dannysmith@users.sourceforge.net>
```

Comment:
nit: I'd put this comment in the first line, so it's clear that this affects the entire file, and not just this function (even though the blank line hints at this).

Reply:
Done.